### PR TITLE
(Feat) Changed unit tests to match recent refactor to GameController. Small refactor to GameController

### PR DESCRIPTION
**Pull request by realmike33**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,9 @@ bower_components/
 
 build/
 dist/
+.DS_Store
+.sass-cache
```
Comment (ryanjlee) [inline .gitignore:10]: Any reason why .DS_Store and .sass-cache is added a second time?

File: client/index.html [hunk 2]
```diff
@@ -7,6 +7,14 @@
 </head>
 
 <body>
+  
+
+  <ui-view></ui-view>
+
```
Comment (ryanjlee) [inline client/index.html:13]: What was the reason for moving this up and adding all the white space? Wasn't it working fine before?

File: .gitignore (hunk 1 above)
Comment (realmike33) [inline .gitignore:10]: Dunno. Can delete out. Didn't notice it.

File: client/index.html (hunk 2 above)
Comment (realmike33) [inline client/index.html:13]: Not test wise. Was receiving weird errors, so decided to move some things around. Everything still works. Added white space mainly for quick visual as to where things started and ended. Can edit if needed.

File: .gitignore (hunk 1 above)
Comment (ryanjlee) [inline .gitignore:10]: Yeah, go ahead and delete and commit it, before we forget about it.

File: client/index.html (hunk 2 above)
Comment (ryanjlee) [inline client/index.html:13]: Ok, that works, but one thing you should probably fix is move the script tags into the body at the end. It's supposed to be best practice.

File: .gitignore (hunk 1 above)
Comment (realmike33) [inline .gitignore:10]: Done

File: client/index.html (hunk 2 above)
Comment (realmike33) [inline client/index.html:13]: and done.

**Comment by ryanjlee** [comment]
@realmike33 Was the changes added to this pull request? I'm not seeing them.

**Comment by ryanjlee** [comment]
Oh you just closed it. Got it.

**Pull request closed without merging**